explain-aggregation.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Explain Statements Using Aggregation
```

Comment:
```suggestion
title: EXPLAIN Statements Using Aggregation
```

Reply:
I think it should be Explain without caps, since in this context it is being used as a verb? I have it as Explain in the other docs.

Reply:
Yes

Reply:
Right, it should probably be "with aggregation (functions)". I would like to keep it consistent between the documents in this section - can we change it back to single words?  Aggregation, Indexes, Views, Index Merge. This will mean the title "Views" is duplicated by another document in a different section.

Reply:
If we use single-word titles, the titles are not specific enough to reflect the document content, and readers must read the document text to get it. If these documents are in an `EXPLAIN` series, it would be better to make each one self-contained as much as possible. `Explain Statements That Use Aggregation` LGTM, because `with` can still be ambiguous.

Reply:
This is fine as long as we use long descriptions on the title, and short on the TOC. The long descriptions are very wordy otherwise:
```
Understanding the Query Execution Plan
  Overview
  Explain Walkthough
  Explain Statements That Use Indexes
  Explain Statements That Use Joins
  Explain Statements That Use Subqueries
  Explain Statements That Use Views
  Explain Statements That Use Index Merge
  Explain Statements That Use Aggregation
  Explain Statements That Use Partitions
```

I prefer:

```
Understanding the Query Execution Plan
  Overview
  Explain Walkthough
  Indexes
  Joins
  Subqueries
  Views
  Index Merge
  Aggregation
  Partitions
```